region/wreckedship/main.json

```diff
@@ -666,6 +681,51 @@
                     "It is possible to use a single power bomb to break the bomb blocks going left to the item, right to the door, and shot blocks going down.",
                     "Place the power bomb against the right wall at the top of the stairs between the three obstacles."
                   ]
+                },
+                {
+                  "name": "G-mode Overload PLMs",
+                  "notable": false,
+                  "requires": [
+                    "Morph",
+                    {"comeInWithGMode": {
+                      "fromNodes": [2,3,4,5],
+                      "mode": "any",
+                      "artificialMorph": false
+                    }}
+                  ],
+                  "note": [
+                    "Use the camera scroll blocks just right of the bomb blocks, in order to overload PLMs and go through them.",
+                    "Cancel g-mode, morph and move back towards the bomb blocks in order to fix the camera."
+                  ]
+                },
+                {
+                  "name": "G-mode Morph to Missiles While Blind",
+                  "notable": false,
+                  "requires": [
+                    {"or": [
+                      {"comeInWithGMode": {
+                        "fromNodes": [2,4,5],
+                        "mode": "any",
+                        "artificialMorph": true
+                      }},
+                      {"and": [
+                        {"comeInWithGMode": {
+                          "fromNodes": [3],
+                          "mode": "any",
+                          "artificialMorph": true
+                        }},
+                        {"or": [
+                          "h_canArtificialMorphIBJ",
+                          "h_canArtificialMorphSpringBallBombJump"
+                        ]}
+                      ]}
+                    ]},
+                    "canOffScreenMovement"
+                  ],
+                  "note": [
+                    "Use the camera scroll blocks just right of the bomb blocks, in order to overload PLMs and go through them.",
+                    "Exit g-mode and travel to the item and back while off screen."
```

Comment:
You can't pickup the item or see what it is with plms overloaded.

Reply:
Right, but it will still be off screen, unless you have morph. 

I'm planning on adding gmode immobile and g-mode setup strats in the next PRs.

Reply:
Ok, I didn't see the difference between strats.  The other one has morph.

The next question is how you avoid spikes blind.  The platform enemies won't move.  And I think the conveyers aren't active either in gmode.

Reply:
There's a good chance regular 9->8 needs `canCarefulJump` with power on, something to test.

Reply:
Dodging the spikes will be very annoying. I am thinking of having `off screen movement` be relatively complex stuff, and possibly be in Insane difficulty on Map Rando. It's either that or have many notable off screen strats. Even Basement right to left is challenging while off screen

Reply:
Yeah that's fair.  Even simple offscreen movement is really hard.